h2o-core/src/main/java/water/H2O.java

```diff
@@ -2171,7 +2180,7 @@ public static void main( String[] args ) {
     ArrayList<String> args2 = new ArrayList<>(Arrays.asList(args));
     for( Object p : System.getProperties().keySet() ) {
       String s = (String)p;
-      if( s.startsWith("ai.h2o.") ) {
+      if( s.startsWith("ai.h2o.") && IGNORED_PROPERTIES.matcher(s).matches()) {
```

Comment:
I am having a hard time understanding this. If it starts with "ai.h2o." and matches "ignored properties" pattern? Is that right?

Reply:
seems it shoud be `! IGNORED_PROPERTIES.matcher(s).matches()`

Reply:
I think the logic works but should be probably called something else than IGNORED_PROPERTIES, maybe "ARGUMENT_PROPERTIES_PATTERN"

see
    assertTrue(H2O.IGNORED_PROPERTIES.matcher("ai.h2o.hdfs_config").matches());